### PR TITLE
Setup Grafana provisioning for docker-compose setup

### PR DESCRIPTION
**Pull request by hagen1778**
PR provides provisioning for Grafana in docker-compose setup. See README for details

### CODE REVIEW

File: deployment/docker/provisioning/dashboards/dashboard.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
+
+providers:
+- name: Prometheus
+  orgId: 1
+  folder: ''
+  type: file
+  options:
+    path: /etc/grafana/provisioning/dashboards
```
Comment (tenmozes) [inline deployment/docker/provisioning/dashboards/dashboard.yml]: missing new line in the end of file

File: deployment/docker/provisioning/datasources/datasource.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      type: prometheus
+      access: proxy
+      url: http://prometheus:9090
+      isDefault: false
+
+    - name: VictoriaMetrics
+      type: prometheus
+      access: proxy
+      url: http://victoriametrics:8428
+      isDefault: true
```
Comment (tenmozes) [inline deployment/docker/provisioning/datasources/datasource.yml]: missing new line in the end of file

File: deployment/docker/README.md [hunk 3]
```diff
@@ -2,8 +2,32 @@
...
+* `--opentsdbListenAddr=:4242`
+* `--httpListenAddr=:8428`
+
+##### Prometheus
+
+To access service open following [link](http://localhost:9090).
+
+Prometheus is already configured to use VictoriaMetrics as remote storage.
+
+##### Grafana
```
Comment (tenmozes) [inline deployment/docker/README.md]: please return Note string, for non-docker users that can be not obvious

Comment (hagen1778) [reply deployment/docker/README.md]: VM datasource is provided by provisioning. No need to specify it manually

**Pull request merged**